frontend/src/components/Credentials/GCredentialName.vue

```diff
@@ -28,41 +28,16 @@ SPDX-License-Identifier: Apache-2.0
       <v-card>
         <g-secret-details-item-content
           class="ma-1"
-          :secret="binding._secret"
+          :secret="credential"
```

Comment:
credential might be a workload identity. But component only supports secrets

Reply:
I renamed the component to GCredentialDetailsItemContent.
The component now decides internally if it is able to render the credential details and shows appropriate error information otherwise. WorkloadIdentity support will be added to the component in a future PR.